v2/pkg/engine/resolve/resolve.go

```diff
@@ -19,6 +19,14 @@ import (
 	"github.com/wundergraph/graphql-go-tools/v2/pkg/pool"
 )
 
+const (
+	HearbeatInterval = 5 * time.Second
+)
+
+var (
+	heartbeat = []byte("{}")
```

Comment:
Currently, we have only one but this is protocol specific. I'd give it a proper name prefix.

Reply:
Makese sense. Do you think `Heartbeat()` should take a data parameter, like `Update`?

Reply:
The callee of `hearbeat` don't know about the type of the subscription. I'd ensure everything is well commented, add a prefix and call it done 😃